indico/modules/events/persons/__init__.py

```diff
@@ -32,3 +32,17 @@
 def _sidemenu_items(sender, event, **kwargs):
     if event.can_manage(session.user):
         return SideMenuItem('persons', _('Roles'), url_for('persons.person_list', event), section='reports')
+
+
+@signals.get_placeholders.connect_via('event-persons-email')
+def _get_placeholders(sender, person, event, register_link=True, **kwargs):
```

Comment:
I'd really default to `False` and only set it to `True` in cases where you need it.